src/firmware_new/boards/frankie_v1/io/infineon_TLE5009_E1000.h

```diff
@@ -0,0 +1,66 @@
+#pragma once
+
+#include <math.h>
+/**
+ * This file is an abstraction around the Infineon TLE5009 E1000 GMR angle sensor
+ *
+ * Datasheet:
+ * https://www.infineon.com/dgdl/Infineon-TLE5009_FDS-DataSheet-v01_01-en.pdf?fileId=db3a304330f686060131421d8ddd56b0
+ */
+
+typedef struct InfineonTLE5009E1000 InfineonTLE5009E1000_t;
```

Comment:
Could we maybe rename this to `InfineonTLE5009E1000AngleSensor`? Similar to how we have `AllegroA3921MotorDriver`

Reply:
Done 👍